tests/test_compact.py

```diff
@@ -254,23 +254,37 @@ def test_compact(api, utils):
 
     validate_compact_config(api, config_values, rx_device_with_rr)
     print("Starting all protocols ...")
-    ps = api.protocol_state()
-    ps.state = ps.START
-    api.set_protocol_state(ps)
+    # ps = api.protocol_state()
```

Comment:
better remove uncomment area which we already remove from model.

Reply:
removed.